Cargo.toml

```diff
@@ -134,7 +134,7 @@ etcd-client = "0.14"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "f0913f179ee1d2ce428f8b85a9ea12b5f69ad636" }
+greptime-proto = { git = "https://github.com/firefantasy/greptime-proto.git", rev = "9f8ae480da8fa14f23cac61e45523cad0f2fac65" }
```

Comment:
Please draft a pull request to [greptime-proto](https://github.com/GreptimeTeam/greptime-proto) and we're good to merge this PR once the proto is merged. Thanks @linyihai